index.html

```diff
@@ -4,26 +4,57 @@
 	<meta charset="utf-8">
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<title>宵街映写 -プロジェクションナイト- 2019</title>
-	<link rel="stylesheet" type="text/css" href="ress.min.css">
-	<link rel="stylesheet" type="text/css" href="public.css">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<link rel="stylesheet" type="text/css" href="server/public/ress.min.css">
+	<link rel="stylesheet" type="text/css"  media="screen"  href="server/public/public.css">
 	<script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/2.1.3/TweenMax.min.js"></script>
 	<script src="https://cdnjs.cloudflare.com/ajax/libs/socket.io/2.2.0/socket.io.js"></script>
-	
+	<script src="https://code.createjs.com/1.0.0/createjs.min.js"></script>
+	<script src="server/public/top.js"></script>
+	<script src="server/public/receive.js"></script>
+
 </head>
 
 <body>
-	<div class="wait" id="js-wait"></div>
 
-	<div id="js-log"><!-- ここに投票内容が出ます --></div>
+	<div id="topPage">
+	      <div class="topLogo">
+	              <img src="server/public/images/1.png" id="maru">
+	                 <img src="server/public/images/topLogo.png"id="topImg" >
+	            </div>
+
+
+	      <div id="topPageContents">
+
+	        <div id="button00" class="rectButton" style="pointer-events: auto">
+	            <p class="buttonText"  onclick="pushButton(0)">start</p></div>

```

Comment:
可読性を下げ、コードが複雑になるので onClick 属性はHTMLで使用しないでください。
下記を参考にjsは別ファイルに分離してください。

```
var buttonText = document.querySelector('#buttonText');
buttonText.addEventListener('click', function() {
  // クリックされた時の処理
});
```

Reply:
onClick属性ですが、4つあるボタンの動き制御するコードを簡略化するためにjsでボタンを取得して使い分けていますが、その辺も参考のコードを例に書き直すべきでしょうか。
onClick属性での制御する方法を変えるとしたら大幅な書き換えが必要ですがどう致しましょう。
```suggestion
　//ボタンの取得
    for(var i= 0;i < numButton;i++){button[ i ]= document.getElementById('button'+ i);}
    
    function pushButton ( targetButton )
      {
        var clickedButton;               //clickedButtonをつかうよ（ローカル変数）
        allButtonPermissionClickEvent();　//すべてのボタンのクリックイベントを無効にするよ
        //flashButton( targetButton );    //押したボタンが光る
        console.log( targetButton );     //押したボタンがコンソールに出力されるよ
        switch(targetButton)
        {


          case 0:
          case 1:
          clickeButton = targetButton;
          setTimeout( function(){ expandRect( targetButton )}, 300 );
          console.log("case 1");
          break;

          case 2:
          clickedButton = 1;
          setTimeout( function(){ deleteTop( clickedButton ) }, 600 );
          console.log("case 2");
          break;

          case 3:
          setTimeout( reduceRect, 600 );
          console.log("case 3");
          break;

          default:
          break;

        }
    
```

Reply:
react や vue ではないのでHTMLに onClick 属性でロジックを組み込むべきではないと思います。
もちろんケースバイケースですが、複数人で開発する以上、フォーマットを揃えておいた方が後々の書き換え（工数）はトータルでは減ると考えています。

Reply:
わかりました。書き換えについてなんですがquerySelector()メゾットが使って、button0~3までの分の下記のコードを書く感じなのでしょうか。
```
var buttonText = document.querySelector('#buttonText');
buttonText.addEventListener('click', function() {
  // クリックされた時の処理
});
```

Reply:
これが絶対に正しいって訳ではないけど、現状のコードをなるべく残したいならこんな感じかな。 [https://jsfiddle.net/tqnd9hcv/2/](https://jsfiddle.net/tqnd9hcv/2/) リンク先確認してみてください。